scripts/build.py

```diff
@@ -156,29 +226,99 @@ def process_js(source_path, target_path):
     _minify(source_path, target_path)
 
 
-def build_minified_third_party_libs(output_directory):
+def get_dependency_filepaths(dependency, filepaths):
+    """Extracts dependency filepaths into filepaths object.
+
+    Args:
+        source_paths: dict(str, str). Dependency information object.
+        filepaths: dict(str, list(str)). Filepaths object with three lists each
+            for different file type (js, css, fonts).
+    """
+    if 'targetDir' in dependency:
+        dependency_dir = dependency['targetDir']
+    else:
+        dependency_dir = (
+            dependency['targetDirPrefix'] + dependency['version'])
+    if 'bundle' in dependency:
+        dependency_bundle = dependency['bundle']
+        for css_file in dependency_bundle.get('css', []):
+            filepaths['css'].append(
+                os.path.join(
+                    THIRD_PARTY_STATIC_DIR, dependency_dir, css_file))
+        for js_file in dependency_bundle.get('js', []):
+            filepaths['js'].append(
+                os.path.join(
+                    THIRD_PARTY_STATIC_DIR, dependency_dir, js_file))
+        if 'fontsPath' in dependency_bundle:
+            for extension in FONT_EXTENSIONS:
+                filepaths['fonts'].append(
+                    os.path.join(
+                        THIRD_PARTY_STATIC_DIR, dependency_dir,
+                        dependency_bundle['fontsPath'], extension))
+
+
+def get_dependencies_filepaths():
+    """Extracts dependencies filepaths from manifes.json file into dictionary
+
+    Returns:
+        dict(str, list(str)). Filepaths object with three lists each for
```

Comment:
See above, there isn't really such a thing as a "Filepaths object". Might want to reword accordingly.

Reply:
Done.

Reply:
Not done?

Reply:
Oh, missed that one. Done.